packages/emails/templates/attendee-scheduled-email.ts

```diff
@@ -70,6 +74,10 @@ export default class AttendeeScheduledEmail extends BaseEmail {
   }
 
   protected getNodeMailerPayload(): Record<string, unknown> {
+    const clonedCalEvent = cloneDeep(this.calEvent);
```

Comment:
is it necessary to clone it?

Reply:
Can we pinpoint where the mutation is happening in address it there?